src/Compilers/Core/Portable/SourceGeneration/GeneratorDriverAnalyzerOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis
+{
+    //PROTOTYPE: we'll wire this up to the project options when we figure that part out
+    internal class GeneratorDriverAnalyzerOptions : AnalyzerConfigOptionsProvider
+    {
+        public override AnalyzerConfigOptions GetOptions(SyntaxTree tree)
+        {
+            return new CompilerAnalyzerConfigOptions(System.Collections.Immutable.ImmutableDictionary<string, string>.Empty);
```

Comment:
Given the double use of the value consider adding a `using` for `System.Collections.Immutable`